CMakeLists.txt

```diff
@@ -81,6 +81,24 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc64|powerpc64)")
     if(${OQS_DIST_BUILD})
         set(OQS_DIST_PPC64_BUILD ON)
     endif()
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc|powerpc)")
+    message(WARNING "There is currently no CI for: " ${CMAKE_SYSTEM_PROCESSOR})
+    # CMake uses uname to derive CMAKE_SYSTEM_PROCESSOR value, so on Darwin
+    # the value is identical for ppc and ppc64. To have the right build arch
+    # in 64-bit case, we use CMAKE_OSX_ARCHITECTURES.
+    if(APPLE AND CMAKE_OSX_ARCHITECTURES STREQUAL "ppc64")
+        set(ARCH "ppc64")
+        set(ARCH_PPC64 ON)
+        if(${OQS_DIST_BUILD})
+            set(OQS_DIST_PPC64_BUILD ON)
+        endif()
+    else()
```

Comment:
Is this `else` correct? It seems to set the following also for non-Apple platforms...

Reply:
@baentsch I believe yes, because other systems using `ppc64` should be taken care of by the case above. So at this point we are left with `ppc32` – plus an odd case of macOS, which supports both ppc and ppc64 in the same OS (while on *BSD the arch of the OS itself must be either ppc or ppc64).

Reply:
Thanks for that rationale @baentsch : Makes sense.